jdaviz/configs/default/plugins/gaussian_smooth/tests/test_gaussian_smooth.py

```diff
@@ -1,21 +1,18 @@
 import pytest
-from jdaviz import Application
+from astropy.utils.exceptions import AstropyUserWarning
 from specutils import Spectrum1D
 
-from jdaviz.configs.default.plugins.gaussian_smooth.gaussian_smooth import GaussianSmooth
 
-
-def test_linking_after_spectral_smooth(spectrum1d_cube):
-
-    app = Application(configuration="cubeviz")
+def test_linking_after_spectral_smooth(cubeviz_helper, spectrum1d_cube):
+    app = cubeviz_helper.app
     dc = app.data_collection
-    app.add_data(spectrum1d_cube, 'test')
-    app.add_data_to_viewer('flux-viewer', 'test')
+    cubeviz_helper.load_data(spectrum1d_cube, data_label='test')
+    spec_viewer = cubeviz_helper.app.get_viewer('spectrum-viewer')
 
     assert len(dc) == 1
 
-    gs = GaussianSmooth(app=app)
-    gs.dataset_selected = 'test'
+    gs = cubeviz_helper.plugins['Gaussian Smooth']._obj
+    gs.dataset_selected = 'test[FLUX]'
```

Comment:
There is slight behavior change when you use the proper route to load the data into Cubeviz. I think this is more correct as a test because this is how user will actually load it.